src/lib/isCnpj.js

```diff
@@ -0,0 +1,23 @@
+export default function isCnpj(cnpj) {
```

Comment:
I would rename this validator to something more generic and add a country code to this. Something like `isCompanyRegistration` could work. Since other countries might have Chambers of Commerce that we can validate in the future

Reply:
I would like to second this, it makes sense to include the CNPJ in a more generic validator, that can be extended with other number schemes in the future